ha/fault_tolerance/fault_tolerance_driver.py

```diff
@@ -14,42 +14,57 @@
 # For any questions about this software or licensing,
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 
+"""
+   Set of periodically executable routines which accepts fault events from
+   message bus and takes further action on it
+"""
+
 
 import time
 
 from cortx.utils.conf_store import Conf
 from cortx.utils.log import Log
+from cortx.utils.message_bus import MessageConsumer
 
-from ha import const
 from ha.core.config.config_manager import ConfigManager
+from ha import const
 from ha.k8s_setup.const import _DELIM
+from ha.core.event_analyzer.filter.filter import K8SFilter
 
 
-class FaultToleranceDriver:
+class FaultTolerant:
     """
-    Driver class which will poll contineously with a specific
-    time gap and analyzes the alert condition and it gets
-    notified to event manager
+    Module responsible for consuming messages from message bus,
+    further analyzes that event and publishes it if required
     """
     def __init__(self, poll_time=10):
+        """Init method"""
+        self._poll_time = poll_time
```

Comment:
why is poll time needed ? wont the callback function get called anytime a new message comes on messagebus ?

Reply:
earlier it was for prometheus poll time (means it should be equal). But now, its just the time and will be used in time.sleep now
And I have changed this to use callback function

Reply:
Rename to wait_time/yield_time